.github/workflows/community.yml

```diff
@@ -1,27 +1,26 @@
 on:
   fork:
   push:
-    branches: [ develop ]
+    branches: [develop]
   issues:
-    types: [ opened ]
+    types: [opened]
   issue_comment:
-    types: [ created ]
+    types: [created]
   pull_request_target:
-    types: [ opened ]
+    types: [opened]
   pull_request_review_comment:
-    types: [ created ]
+    types: [created]
 
 jobs:
-
   welcome:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
       - uses: EddieJaoudeCommunity/gh-action-community/src/welcome@main
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
-          issue-message: '<h1>It''s great having you contribute to this project</h1> Feel free to raise an <strong>Issue</strong>! Welcome to the community :nerd_face:'
-          pr-message: '<h1>It''s great having you contribute to this project</h1> Feel free to create a <strong>Pull Request</strong>! Welcome to the community :nerd_face:'
+          issue-message: "<h1>It's great having you contribute to this project</h1> Feel free to raise an <strong>Issue</strong>! Welcome to the community :nerd_face:"
```

Comment:
These didnt change back to single quotes, but in other `yaml` files it did, does something need to be run again?

Reply:
This is an issue with prettier maybe as it does work with .yaml but not with .yml . The issue was raised [here](https://github.com/prettier/prettier-vscode/issues/278), but I couldnt find a workaround. Do I change them manually for now?

Reply:
I wonder prettier-vscode has some problem with it, I am running them on all files. As a workaround, I am reverting the commas here.

Reply:
Ok thank you 👍 . I would suggest, reverting the commas here and run it again and see if it changes it again or if it leaves it alone.

Thank you

Reply:
> Ok thank you 👍 . I would suggest, reverting the commas here and run it again and see if it changes it again or if it leaves it alone.
> 
> Thank you 

It leaves them as it is.